### ISSUE TITLE
There is no way to evaluate expression only once

### ISSUE THREAD

**Issue by elijahiuu** [vuejs/vue#7072]
### Version
2.5.3

### Reproduction link
[https://jsfiddle.net/elijahww/gojaz8cx/](https://jsfiddle.net/elijahww/gojaz8cx/)

### Steps to reproduce
There is currently no way to execute an expression only once. My use-case is simple i18n translation.
<span>{{ $myGlobali18service('foo.bar.hello') }}</span>

It would be nice to have some syntax to tell {{ }} interpolator to evaluate only once, like you can do on directives. Example: <span v-my-global-i18-service=" 'foo.bar.hllo' "></span>   and ensure it is executed only within the bind() function, and not update(). However, it is verbose and needs single quotes within double quotes and looks pretty messy in my templates. 

Are there any other solutions? 

### What is expected?
execute specialized {{}} interpolations only once

### What is actually happening?
{{}} execute on every update

**Comment by posva** [comment]
You can also memoize calls: https://jsfiddle.net/cv8wjj0c/
You can check https://github.com/kazupon/vue-i18n, I don't know if the memoize values but people seem to enjoy using it 🙂

**Comment by elijahiuu** [comment]
Good idea, @posva thank you! vue-i18n code suffers from the same thing, they could use your approach :)

[http://kazupon.github.io/vue-i18n/en/directive.html#t-vs-v-t](http://kazupon.github.io/vue-i18n/en/directive.html#t-vs-v-t)
> Cons
> $t is executed everytime when re-render occures, so it does have a translation costs.

**Comment by posva** [comment]
Keep in mind my example is very simple, you need to invalidate the cache somehow if you load new translations and probably handle other edge cases
